ironfish-cli/src/commands/peers/index.ts

```diff
@@ -19,10 +19,7 @@ export class ListCommand extends IronfishCommand {
   static flags = {
     ...RemoteFlags,
     ...tableFlags,
-    sort: {
-      ...sort,
-      exclusive: ['follow'],
-    },
+    sort,
```

Comment:
I think you can just replace L21 and 22 with `...CliUx.ux.table.flags()` and can remove the de-structure on L14 since we're not modifying the `sort` key anymore

Reply:
Done